test/test_um2netcdf.py

```diff
@@ -397,25 +397,30 @@ class DummyCube:
     Imitation iris Cube for unit testing.
     """
 
-    def __init__(self, item_code, var_name=None, attributes=None, units=None):
+    def __init__(self, item_code, var_name=None, attributes=None,
+                 units=None, coords=None):
         self.item_code = item_code
         self.var_name = var_name or "unknown_var"
         self.attributes = attributes
         self.units = None or units
```

Comment:
I don't think so. 

Another general comment about using:
```py
var = something_might_be_None or something_else
```

I usually avoid declaring variables using `or` if I only want to declare another value for `None` values of `something_might_be_None`. 
This because if there are cases when `something_might_be_None` might be `0` (on purpose), `var` would still be assigned to `something_else`.

In this case it should never be a problem because I don't think `var_name` or `units` can ever be 0, but as a general rule of thumb I would always use the more specific:
```py
var = something_might_be_None if something_might_be_None is not None else something_else
```

Reply:
This is probably cruft from the dev process, gradually expanding test objects on the fly & that CPython requires default args for all args after a default arg.

Reply:
Fixed in 4144b41f.